src/main/java/io/opentracing/contrib/global/concurrent/TracedCallable.java

```diff
@@ -0,0 +1,68 @@
+package io.opentracing.contrib.global.concurrent;
+
+import com.google.common.io.Closer;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer.SpanBuilder;
+import io.opentracing.contrib.global.GlobalTracer;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Convenience {@link Callable} wrapper that will execute within a new {@link Span} if an
+ * {@link #withOperationName(String) operationName} is provided.<br>
+ * If no operationName is provided, the call will be executed without starting a new Span.
+ */
+public class TracedCallable<T> implements Callable<T> {
+
+    private final Callable<T> delegate;
+    private String operationName = "";
+    private SpanContext parentContext = null;
+
+    protected TracedCallable(Callable<T> delegate) {
+        if (delegate == null) throw new NullPointerException("Callable delegate is <null>.");
+        this.delegate = delegate;
+    }
+
+    public static <T> TracedCallable<T> of(Callable<T> delegate) {
+        return new TracedCallable<T>(delegate);
+    }
+
+    public TracedCallable<T> withOperationName(String operationName) {
+        this.operationName = operationName;
+        return this;
+    }
+
+    public TracedCallable<T> asChildOf(SpanContext parent) {
+        this.parentContext = parent;
+        return this;
+    }
+
+    protected Span startNewSpan() {
+        SpanBuilder spanBuilder = GlobalTracer.get().buildSpan(operationName);
+        if (parentContext != null) spanBuilder = spanBuilder.asChildOf(parentContext);
+        return spanBuilder.start();
+    }
```

Comment:
hm, so it seems contrary to the comments in the `traced()` methods, there is always a child span started. Once again, I question why this library needs to be so opinionated. Why not start with a simple `traced(span, callable)`? If people actually need any of this extra help, it is easy to add it in the future.

It's not even the overhead of always creating a span that bothers me, it's the fact that you are using `childOf` reference type. I can quite easily imagine that an async task served by a callable should be a FOLLOWS_FROM reference type, because we don't know whether the parent depends on the results from this callable.

The main issue with spans for async tasks is making sure the right span is accessible once you cross the thread boundary. That can only be solved when you have context propagation mechanism, i.e. in `java-activespan` library. 

My recommendation is to remove all runnable/callable functionality from this library. Starting a new span is the prerogative of OpenTracing API. If people want to start a new span inside runnable, they can use OT for that, there's no need to reinvent it here.